src/pvecontrol/__init__.py

```diff
@@ -7,16 +7,15 @@
 import confuse
 import urllib3
 import logging
-from prettytable import PrettyTable
-from collections import OrderedDict
-from humanize import naturalsize
 
 from pvecontrol.node import PVENode
 from pvecontrol.node import NodeStatus
 from pvecontrol.vm import PVEVm
 from pvecontrol.vm import VmStatus
 from pvecontrol.task import PVETask
+from pvecontrol.utils import _print_tableoutput, _filter_keys
```

Comment:
In regard to [PEP 8](https://peps.python.org/pep-0008/#descriptive-naming-styles), underscore prefixed names suggest an internal/private use. You should rename those I think.

> _single_leading_underscore: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.

Reply:
Yep _ means "local"  which does not exists in python :/